performance_manager/lib/s3_utils.py

```diff
@@ -17,11 +17,21 @@ def _get_pyarrow_table(
     """
     internal function to get pyarrow table from parquet file(s)
     """
+
     active_fs = fs.S3FileSystem()
+
     if isinstance(filename, list):
-        to_load = [f.replace("s3://", "") for f in filename]
+        if filename[0].startswith("s3://"):
+            to_load = [f.replace("s3://", "") for f in filename]
+        else:
+            active_fs = fs.LocalFileSystem()
+            to_load = filename
     else:
-        to_load = [filename.replace("s3://", "")]
+        if filename.startswith("s3://"):
+            to_load = [filename.replace("s3://", "")]
+        else:
+            active_fs = fs.LocalFileSystem()
+            to_load = [filename]
```

Comment:
This `fs.LocalFileSystem()` logic breaks our `--seed` flag path loads. If `fs.LocalFileSystem()` is only used for testing maybe we can handle with and ENV flag that is set just for testing??

Reply:
i think the better method is to do a monkey patch here.